src/components/Chip/components/Badge/Badge.tsx

```diff
@@ -2,14 +2,14 @@ import React from 'react';
 import { generateTestDataId } from 'utils/helpers';
 import { TestId } from 'utils/types';
 
-import { Props } from '../../Chip.types';
+import { ChipProps } from '../../Chip.types';
 import { badgeStyle } from './Badge.style';
 
 type TestProps = {
   dataTestId?: TestId;
 };
 
```

Comment:
Isn't this type already exported somewhere else in the repo? Is this necesarry?

Reply:
yes, but this is not in the scope of the PR